### ISSUE TITLE
:button_css: option seem to not only affect the current button.

### ISSUE THREAD

**Issue by Carreau** [jupyterlite/jupyterlite-sphinx#127]
my understanding was that :

```
.. try_examples::
    :button_css:
        background-color: red;
```

Would only make the *current* button red, though it seem to affect at least all the buttons on the page.

Maybe related to #122, but I suggest we have a single `:example-class: <str>`, that can be styled by the custom css. Even the button-top/button-bottom can be styled using custom flex-column/flex-reverse-column.

**Comment by Carreau** [comment]
Yeah, I think an `:example-class:` or `:example-id:` as technically if something is unique in css you should use ID, but I think class + custom css should be able to do anything if we put the class on the outer try-example div.

**Comment by Carreau** [comment]
I think this is closed by #129.
